### ISSUE TITLE
Removing media folder should stop media library scan.

### ISSUE THREAD

**Issue by bpcomp13** [popeen/Booksonic-Air#20]
## Problem description

When attempting to add a media folder, if you add a top level folder suck as / there does not appear to be any way to stop a media folder scan. Deleting the media folder should stop media scanning of that folder.

### Steps to reproduce

1. Add a media folder with folder location of /
2. Click the scan media folders now

## System information

 * **Airsonic version**: 2009.1.0 – September 15, 2020, based on Airsonic 8b28366 from Docker
 * **Operating system**: Ubuntu 16.04 LTS
 * **Java version**: java 1.8.0_265
 * **Proxy server**: None
 * **Client**: Chrome 85.0.4183.102 and Firefox 80.0.1
 * **Language**: English

**Comment by popeen** [comment]
Yeah, having a way to stop the scan sounds like a really good idea
